mypy/dmypy.py

```diff
@@ -93,7 +105,7 @@ def __init__(self, prog: str) -> None:
                help="Server shutdown timeout (in seconds)")
 p.add_argument('flags', metavar='FLAG', nargs='*', type=str,
                help="Regular mypy flags (precede with --)")
-
+p.add_argument('--options-file', help=argparse.SUPPRESS)
```

Comment:
If it is impossible to pass the `Options` as a pickled object anyway, perhaps I could pass an `Options.snapshot` (which is just a dict image of the class) instead of the class itself. As for passing the pickled object via an argument, that seems plausible, I will experiment.